.changeset/dull-singers-decide.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/core': patch
+'@twilio-paste/data-grid': patch
+---
+
+include supporting types and documentation for existing colSpan prop
```

Comment:
```suggestion
[Data Grid] Include supporting types and documentation for existing colSpan prop
```
suggestion (non-blocking): we usually add the package name to the changeset message